src/modules/wallet/components/overview/overview.test.js

```diff
@@ -26,6 +27,7 @@ jest.mock('@account/hooks');
 jest.mock('@block/hooks/queries/useBlocks');
```

Comment:
@eniolam1000752 `useBlocks` is no longer in use and so the mock should be removed